cloud_functions/api_router/main.py

```
@@ -0,0 +1,450 @@
+                            "fitnessllm_dataplatform.task_handler",
+                            "full_etl",
+                            f"--uid={payload['uid']}",
+                            "--data_source=STRAVA",
+                        ]
+                    }
+                ],
+            }
+        }
+
+        # Make the request
+        response = requests.post(url, json=new_payload, headers=headers)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)